test/test-template-service.ts

```diff
@@ -73,13 +73,18 @@ describe("TemplateService Api", () => {
             templateService.getTemplates().then((templates: any) => {
                 should.exist(templates);
                 templates.should.be.an("array");
+                templates.length.should.be.above(0);
                 if (templates.length) {
                     for (let ti = templates.length - 1; ti >= 0; ti--) {
```

Comment:
I think that this for loop is obsolete. We should be able to check properties like we do in the test-template-backup.
E.g: 
templates.should.all.have.property("name");
templates.should.all.have.property("description");
templates.should.all.have.property("displayName");
etc

Reply:
will make better tests once #33 is merged

Reply:
@lini #33 merged